ui/shared/components/panel/variants/Pathogenicity.jsx

```diff
@@ -96,6 +110,7 @@ const Pathogenicity = React.memo(({ variant, showHgmd }) => {
       href: clinvarUrl(clinvar),
       goldStars: clinvar.goldStars,
       popup: clinvar.version && `Last Updated: ${new Date(clinvar.version).toLocaleDateString()}`,
```

Comment:
conditions should be shown on hover for the main label, not the Broad submission label. Sorry for not making that clear in the ticket